classes/02class/exercises/c02-network10/gpavelar/README.md

```diff
@@ -0,0 +1,23 @@
+# c02-network10
+
+## Commands Execution Output
+
+- Commands for creating an ENI with an IP from your private subnet
+
+```bash
+## Create ENI
+aws ec2 create-network-interface --subnet-id subnet-08da4676baf658cb2 --description "my private network interface" --private-ip-address 10.0.1.17
+
+# Output
+NETWORKINTERFACE        ap-southeast-2a my private network interface    interface       06:30:76:8d:ed:e8       eni-03cf44676961d7903   478433196210    10.0.1.17       AIDAW6ZGXHCZC7ESLQHIS   False   True    pending subnet-08da4676baf658cb2        vpc-074c260483405f2bb
+GROUPS  sg-00e6b83ea1dccd737    default
+PRIVATEIPADDRESSES      True    10.0.1.17
+```
+
+- Any extra challenges faced?
+
+> Which groups should I use? --groups parameters is not obligatory
```

Comment:
In this example, you are not associating any SG to your ENI.

A key take away here for any work you'll do in AWS: ENIs are the ones having SGs attached to. (there are a few exceptions)

When you create an EC2 instance, an elastic network interface will be created for it (ENI), and this ENI may have SGs associated. 

Besides that, ENIs are the ones having EIP (elastic IPs) attached to. 

So the info you get in the EC2 screen is basically the info from the ENI :)